### PR TITLE
update message for course applide

### PR DESCRIPTION
**Pull request by Liran333**

### CODE REVIEW

**Comment by VictorZhou123** [comment]
lgtm

**Comment by Liran333** [comment]
rebase

File: config/config.go [hunk 1]
```diff
@@ -90,6 +92,7 @@ func (cfg *Config) ConfigItems() []interface{} {
 		&cfg.Points.Domain,
 		&cfg.Points.Repo,
 		&cfg.Cloud,
+		&cfg.Course.Message,
```
Comment (zengchen1024) [inline config/config.go:95]: s/&cfg.Course.Message,/&cfg.Course,

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: message-server/main.go [hunk 2]
```diff
@@ -144,6 +144,8 @@ func pointsSubscribesMessage(cfg *configuration, topics *mqTopics) error {
 			topics.CompetitorApplied,
 			topics.JupyterCreated,
 			topics.CourseApplied,
+      topics.BigModelTopics.PicturePublicized,
```
Comment (zengchen1024) [inline message-server/main.go:147]: align with above line.
对齐上一行

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**